ConsentViewController/Classes/SourcePointClient/SPNativeUI.swift

```diff
@@ -11,26 +11,57 @@ import UIKit
 @objcMembers class SPNativeFont: NSObject, Codable {
     var fontSize: CGFloat
     var fontWeight: String
-    var color: String?
+    var color: String
     var fontFamily: String
     
-    init(fontSize: CGFloat, fontWeight: String, fontFamily: String) {
+    init(fontSize: CGFloat, fontWeight: String, fontFamily: String, color: String) {
         self.fontSize = fontSize
         self.fontWeight = fontWeight
         self.fontFamily = fontFamily
+        self.color = color
+    }
+    
+    required init(from decoder: Decoder) throws {
+        let style = Constants.UI.StandartStyle()
+        let container = try decoder.container(keyedBy: Keys.self)
+        fontSize = try container.decodeIfPresent(CGFloat.self, forKey: .fontSize) ?? style.font.fontSize
+        fontWeight = try container.decodeIfPresent(String.self, forKey: .fontWeight) ?? style.font.fontWeight
+        fontFamily = try container.decodeIfPresent(String.self, forKey: .fontFamily) ?? style.font.fontFamily
+        color = try container.decodeIfPresent(String.self, forKey: .color) ?? style.font.color
+    }
+    
+    enum Keys: CodingKey {
+        case fontSize, fontWeight, color, fontFamily
     }
 }
 
 @objcMembers class SPNativeStyle: NSObject, Codable {
-    var backgroundColor: String?
-//    let width: Int?
-    var font: SPNativeFont?
-    var onFocusBackgroundColor: String?
-    var onUnfocusBackgroundColor: String?
-    var onFocusTextColor: String?
-    var onUnfocusTextColor: String?
-    var activeBackgroundColor: String?
-    var activeFont: SPNativeFont?
+    var backgroundColor: String
+    var font: SPNativeFont
+    var onFocusBackgroundColor: String
+    var onUnfocusBackgroundColor: String
+    var onFocusTextColor: String
+    var onUnfocusTextColor: String
+    var activeBackgroundColor: String
+    var activeFont: SPNativeFont
```

Comment:
@Nevazhnovu if we're not setting these properties elsewhere, I believe they can be set back to `let`.